packages/grafana-data/src/transformations/transformDataFrame.ts

```diff
@@ -107,3 +118,21 @@ export function transformDataFrame(
 function isCustomTransformation(t: DataTransformerConfig | CustomTransformOperator): t is CustomTransformOperator {
   return typeof t === 'function';
 }
+
+function deepIterate<T extends object>(obj: T, doSomething: (current: any) => any): T;
+// eslint-disable-next-line no-redeclare
+function deepIterate(obj: any, doSomething: (current: any) => any): any {
```

Comment:
@kaydelaney i had to restructure this a bit, since a copy/paste from your PR was not working